src/main/java/org/myrobotlab/service/InverseKinematics3D.java

```diff
@@ -239,9 +240,10 @@ public void publishTelemetry(String name) {
       // - 180 to + 180 ?
       double angle = MathUtils.radToDeg(theta) + l.getOffset();
       angleMap.put(jointName, (double) angle % 360.0F);
+      invoke("publishJointAngles", new AngleData(jointName, (double) angle % 360.0F));

```

Comment:
Angles plural is incorrect here.  AngleData is singular.

Reply:
fixed